instrumentation/action_view/test/opentelemetry/instrumentation/action_view/instrumentation_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+require_relative '../../../../lib/opentelemetry/instrumentation/action_view'
+
+describe OpenTelemetry::Instrumentation::ActionView do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::ActionView::Instrumentation.instance }
+
+  it 'has #name' do
+    _(instrumentation.name).must_equal 'OpenTelemetry::Instrumentation::ActionView'
```

Comment:
I am seeing spans emitted by the ActionView instrumentation appear with: `library.name: OpenTelemetry::Instrumentation::Rails`. Is that a result of [which instrumentation gets used in the railtie](https://github.com/open-telemetry/opentelemetry-ruby/pull/796/files#diff-6a1bb501f356942c4ec7170c3e818300c0509c1e0a6ac890dcfc64f54a6a714aR18)?

Reply:
It is. I think I have an idea on how to fix that though.